packages/git-safe-dependencies/src/isgit.js

```diff
@@ -11,7 +11,8 @@ const isUnsafeUrl = (url) =>
 
 const isGitUrl = (url) => gitUrlRegex.test(url)
 const isGHRepoSpecifier = (specifier) =>
-  specifier.startsWith('github:') || specifier.match(/^[^/]+\/[^/]+/)
+  !!(specifier.startsWith('github:') || specifier.match(/^[^/@]+\/[^/]+$/)) &&
+  !specifier.startsWith('npm:')
```

Comment:
```suggestion
  !specifier.startsWith('npm:') && (specifier.startsWith('github:') || /^[^/@]+\/[^/]+$/.test(specifier))  
```

this defers regex execution until the last step and avoids the need for casting to boolean

- [ ] would prefer a `const ghRepoSpecifierRegex = /.../` akin to `gitUrlRegex`
- [ ] please show example of what the regex is trying to match or otherwise document the `isGHRepoSpecifier` function so we know what it's trying to do